Sources/IO/XML/XMLImageDataWriter/index.js

```diff
@@ -23,6 +23,7 @@ function vtkXMLImageDataWriter(publicAPI, model) {
       WholeExtent: dataObject.getExtent().join(' '),
       Origin: dataObject.getOrigin().join(' '),
       Spacing: dataObject.getSpacing().join(' '),
+      Direction: dataObject.getDirection().join(' '),
```

Comment:
it seems that it will crash if no direction is set on the vtkImageData.

Reply:
@ramonemiliani93 can you please address this issue ? it will crash if no Direction is set on the image data.

Reply:
Sure @finetjul let me fix it 👌

Reply:
in vtk.js vtkImageData will always add a direction so it should not be a problem at write time. The issue was at read time but that has been addressed.

Reply:
Oh, good point, I did not see what was going on in `vtkImageData::extent()`. My bad